lib/views/frontend/spree/checkout/payment/_stripe.html.erb

```diff
@@ -78,16 +78,4 @@
   });
 </script>
 
-<%- if @order.has_checkout_step?('address') -%>
-  <script>
-    Spree.stripeAdditionalInfo = {
-      name: "<%= @order.bill_address.full_name %>",
-      address_line1: "<%= @order.bill_address.address1 %>",
-      address_line2: "<%= @order.bill_address.address2 %>",
-      address_city: "<%= @order.bill_address.city %>",
-      address_state: "<%= @order.bill_address.state_text %>",
-      address_zip: "<%= @order.bill_address.zipcode %>",
-      address_country: "<%= @order.bill_address.country %>"
-    }
-  </script>
-<%- end -%>
+<%= render 'spree/checkout/payment/stripe_additional_info' %>
```

Comment:
Let's pass `order.bill_address` as param here in order to avoid breaking LoD

Reply:
fixing